src/pages/AddMark.css

```diff
@@ -103,3 +103,24 @@ textarea:focus, input:focus{
   width: 100vw;
   height: 100vh;
 }
+
+
+input, textarea {
```

Comment:
These styles style all input fields, so they interfere with all the other input fields. 
Please modify so that this style only applies for the elements in AddMark

By the way I think you also don't need the -moz etc. classes as they should be added by autoprefixer automatically (just inspect in your browser and you should see them)